src/main/java/cn/nukkit/network/protocol/AvailableCommandsPacket.java

```diff
@@ -25,23 +26,23 @@
     public static final int ARG_FLAG_SOFT_ENUM = 0x4000000;
 
     public static final int ARG_TYPE_INT = 1;
-    public static final int ARG_TYPE_FLOAT = 2;
-    public static final int ARG_TYPE_VALUE = 3;
-    public static final int ARG_TYPE_WILDCARD_INT = 4;
-    public static final int ARG_TYPE_OPERATOR = 5;
-    public static final int ARG_TYPE_TARGET = 6;
-    public static final int ARG_TYPE_WILDCARD_TARGET = 7;
-
-    public static final int ARG_TYPE_FILE_PATH = 14;
-
-    public static final int ARG_TYPE_STRING = 29;
-    public static final int ARG_TYPE_BLOCK_POSITION = 37;
-    public static final int ARG_TYPE_POSITION = 38;
-
-    public static final int ARG_TYPE_MESSAGE = 41;
-    public static final int ARG_TYPE_RAWTEXT = 43;
-    public static final int ARG_TYPE_JSON = 47;
-    public static final int ARG_TYPE_COMMAND = 54;
```

Comment:
This breaking change will be solved at https://board.powernukkit.org/youtrack/issue/PN-1000